### PR TITLE
routing for cart 

### PR DESCRIPTION
**Pull request by sudharaD**
resolve the issue #20

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ const app = express()
 const userRoutes = require('./routes/user')
 const productRoutes = require('./routes/product')
 const sectionsRoutes = require('./routes/sections')
+const cartRoutes = require('./routes/cart')
+
```
Comment (hello-work) [inline app.js:8]: can you remove this empty line @sudharaD

Comment (hello-work) [inline app.js]:
```suggestion
```

Comment (sudharaD) [reply app.js]: resolve the comment

**Review by hello-work — APPROVED** [review]
LGTM

**Pull request merged**